--- a/Code/Voxels/GpuVoxelMesher.cs
+++ b/Code/Voxels/GpuVoxelMesher.cs
@@ -4460,7 +4460,7 @@
 
 		public override void RenderSceneObject()
 		{
-			if ( Graphics.LayerType != SceneLayerType.DepthPrepass && Graphics.LayerType != SceneLayerType.Shadow ) return;
+			if ( Graphics.LayerType != SceneLayerType.Shadow ) return;
 			lock ( _owner._renderCameraLock )
 			{
 				var state = _owner._visibilityReadbackState;
